analyzer/consensus/genesis.go

```diff
@@ -46,9 +45,8 @@ func (mg *GenesisProcessor) Process(document *genesis.Document) ([]string, error
 	}
 
 	// Rudimentary templating.
-	chainID := strcase.ToSnake(document.ChainID)
 	for i, query := range queries {
-		queries[i] = strings.ReplaceAll(query, "{{ChainId}}", chainID)
+		queries[i] = strings.ReplaceAll(query, "{{ChainId}}", "chain")
```

Comment:
Should we just replace all the `{{ChainId}}` with `chain` in
```           mg.addRegistryBackendMigrations,
		mg.addStakingBackendMigrations,
		mg.addGovernanceBackendMigrations,```

Reply:
Yes, we should, I was just thinking we'd do it in a later PR. I left this in the description of #343 and forgot to move/copy it here after I split that PR into three:

> A similar-to-QF lightweight templating approach remains in genesis.go. I didn't touch it because we'll majorly clean up its SQL generation (from manually stringifying args to using SQL templates) in an upcoming PR.